pkg/iac/detection/detect.go

```diff
@@ -150,8 +155,7 @@ func init() {
 			return false
 		}
 
-		return (sniff.Parameters != nil && len(sniff.Parameters) > 0) ||
-			(sniff.Resources != nil && len(sniff.Resources) > 0)
+		return len(sniff.Parameters) > 0 || len(sniff.Resources) > 0
```

Comment:
I believe we had the `nil` check to make sure we don't panic I case the fields are actually `nil`. Do we not need that?

Reply:
The zero value of a slice is nil https://go.dev/tour/moretypes/12

Reply:
ah good catch!